framework/UPGRADE.md

```diff
@@ -51,6 +51,12 @@ if you want to upgrade from version A to version C and there is
 version B between A and C, you need to follow the instructions
 for both A and B.
 
+Upgrade from Yii 2.0.60
```

Comment:
It should be 2.0.53.

Also, what is the reason for this jump from `2.0.53` to `2.0.60`?

Reply:
Fixed, i didn't remember that Yii2 doesn't use semver.